samples/go/nomdex/vrange.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/attic-labs/noms/go/types"
+)
+
+type ventry struct {
+	v    types.Value
+	incl bool
+	inf  int8
+}
+
+type vrange struct {
```

Comment:
I think that the "v" in this name is superfluous -- it makes me think it is a _virtual_ range or something. Suggest just _range_. Or maybe _resultRange_ if you want to be more descriptive.